test/__snapshots__/ConfigCacheTestCases.longtest.js.snap

```diff
@@ -22,111 +22,144 @@ body {
 	background: black;
 }
 
-@layer (default) {
+@layer default {
 	body {
 		background: black;
 	}
 }
-@layer (default) {
```

Comment:
It was a bug, we don't need to print `(` and `)` around layer, don't know why I missed it